src/platform_impl/windows/window.rs

```diff
@@ -698,6 +698,14 @@ impl Window {
         Some(self.window_state_lock().current_theme)
     }
 
+    #[inline]
+    pub fn title(&self) -> String {
+        let len = unsafe { GetWindowTextLengthW(self.window.0) };
+        let mut buf = vec![0; len as usize];
```

Comment:
don't we need to account for the `\0` terminator here? (haven't tested it so far)

Reply:
yeah you're correct